src/modern/lib/download.test.ts

```
@@ -1,6 +1,37 @@
 import { afterEach, beforeEach, describe, expect, it } from 'bun:test';
 
-import { triggerBlobDownload } from './download';
+import { sanitizeFilename, triggerBlobDownload } from './download';
```

ESLint

[error] Missing file extension "ts" for "./download"

(import/extensions)